setup.py

```diff
@@ -44,32 +41,39 @@ def package_data(pkg, root_list):
 
 BLOCKS = [
     'mentoring = mentoring:MentoringBlock',
-    'mentoring-dataexport = mentoring:MentoringDataExportBlock',
-]
 
-BLOCKS_CHILDREN = [
     'mentoring-table = mentoring:MentoringTableBlock',
-    'column = mentoring:MentoringTableColumnBlock',
-    'header = mentoring:MentoringTableColumnHeaderBlock',
+    'mentoring-column = mentoring:MentoringTableColumn',
     'answer = mentoring:AnswerBlock',
-    'quizz = mentoring:QuizzBlock',
-    'message = mentoring:MentoringMessageBlock',
-    'tip = mentoring:QuizzTipBlock',
-    'choice = mentoring:QuizzChoiceBlock',
-    'html = mentoring:HTMLBlock',
+    'answer-recap = mentoring:AnswerRecapBlock',
+    'mcq = mentoring:MCQBlock',
+    'rating = mentoring:RatingBlock',
+    'mrq = mentoring:MRQBlock',
+    'mentoring-message = mentoring:MentoringMessageBlock',
+    'tip = mentoring:TipBlock',
```

Comment:
@bradenmacdonald @nedbat I agree that currently the children aren't generic, but they already have a common interface that allows them to be used as children by other XBlocks too. We have started doing just that, reusing them in other XBlocks - I eventually would like to see these children moved to their own repositories, standardizing their interface and used by a variety of XBlocks, rather than being really specific to mentoring. If we name these children like "<mentoring-mcq>", we tie them to mentoring, and will have children of other XBlocks using the mentoring prefix in their XML, even though these XBlocks will have nothing to do with mentoring, except for sharing the same internal interface for handling children. These are implementation details I wouldn't expect to expose to course authors.

Now you're right that there could still be alternative implementations of a <mcq> child, so a prefix does make sense, if only to not appropriate the mcq node name - but don't think that mentoring would be a good prefix here, I'd rather use something that would still make sense in the context of other XBlocks using it. I'm not sure what that prefix would be though - imho it would need to be generic enough to make sense for very different types of XBlocks, while still being meaningful. 

So to not block this PR, I can live with using the mentoring prefix. But I'm curious what you think would be the right approach here, for naming children which are meant to be reused across different XBlocks?